rest_api/simple_supply_rest_api/database.py

```diff
@@ -18,8 +18,12 @@
 
 import aiopg
 import psycopg2
+from psycopg2.extras import RealDictCursor
 
 
+LATEST_BLOCK_NUM = """
```

Comment:
There should be a comment explaining this formatting (I assume there's a good reason for it).

Reply:
I'm using this multi-line string formatting for all of the database queries.  Some of them get long and this format helps the readability. To keep it consistent I used the format for this variable as well. It could be changed to be a regular string if you think that would be more clear.